### DISCUSSION TITLE
GitHub Milestones?

### DISCUSSION THREAD

**Discussion by nogginbox** [dotnet/maui#4246] [General] [answered]
I was looking at the [project milestones progress](/dotnet/maui/milestones) and think the issues assigned to each has become out of step with the real world.

It looks like you've not finished preview 11 yet, but I'm using it, so you must have. In fact it also shows you've not finished preview 9 or 10 either, so it looks like quite a few issues need to be moved to new milestones.

**Comment by hartez** [comment, marked as answer]
Yep. They're a bit out of sync. We're working on that.
